libs/SalesforceSDK/src/com/salesforce/androidsdk/app/SalesforceSDKManager.java

```diff
@@ -289,6 +289,9 @@ public LoginOptions getLoginOptions(String jwt, String url) {
                 loginOptions = new LoginOptions(url, getPasscodeHash(), config.getOauthRedirectURI(),
                         config.getRemoteAccessConsumerKey(), config.getOauthScopes(), null, jwt);
             }
+        } else {
```

Comment:
so this is to fix a scenario where the app is first launched normally, then background and foreground through the link, we are not updating loginOption